### PR TITLE
docs: Fixed some typo and grammatical error in docs

### LINKED ISSUE #158: Docs Grammar and Typos
## Skills
Some of the skills you could see yourself learning & applying as you go along:
- Git/GitHub
- Markdown
- English

## What needs to be done
The docs site is still new so there is bound to be typos and grammatical errors. All english correction pull requests are welcome and greatly appreciated. 

## How to do it
If you see a URL with a typo, you can go to the `content/` directory and go into the corresponding the `.md` markdown file to make changes. For example, if I wanted to fix a typo in https://docs.atsign.com/sdk/flutter/introduction/, the file path would be somewhere in `content/docs/sdk/flutter/`. Sometimes you need to go into a folder and open a `index.md` file.
 
## Support
In case you need any kind of help (such as getting your environment setup or asking questions about our tech) don’t be afraid to ask in our [discord](https://discord.atsign.com). There is a team full of developers ready to jump in and help on whatever your issue may be.

### PR DESCRIPTION
**Pull request by HVbajoria**
This pull request corrects some typos and grammatical mistakes in the docs. Fixes #158 

**- What I did**
I checked the files in the doc and found some typo and grammatical errors and corrected them.

**- How I did it**
I manually checked the files to find if there is any issue present.

**- Description for the changelog**
Fixed typos and grammatical errors

### CODE REVIEW

File: content/docs/reference/synchronization.md [hunk 1]
```diff
@@ -16,18 +16,18 @@ weight: 210 # For single pages, lower is first.
 
 ## Definition
 
-On at the atProtocol, your data is encrypted with your self [encryption](/reference/encryption) key and stored on your device. Periodically, this data is copied securely over to a dedicated cloud server which only you can decrypt and read since you are the only one who owns the private key. No one else, including **Atsign** can read your data. This process is known as synchronization.
+At the atProtocol, your data is encrypted with your self [encryption](/reference/encryption) key and stored on your device. Periodically, this data is copied securely over to a dedicated cloud server which only you can decrypt and read since you are the only one who owns the private key. No one else, including **Atsign** can read your data. This process is known as synchronization.
```
Comment (cpswan) [inline content/docs/reference/synchronization.md]: s/At/With/

File: content/docs/reference/pricing-and-costs.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ weight: 201 # For single pages, lower is first.
...
+Since the atPlatform is fully open-sourced and everyone can contribute, there is no cost for developers! When you make an app on the atPlatform, people that use your apps will not have to pay any fees for infrastructure or storage! The way that **Atsign** generates revenue is by using our special identifier which is known as an atSign. An atSign can come in many flavors with corresponding prices ranging from free, $10, $100, $1000, $5000. If you opt-in for the vanity atSign the prices increase as the atSign of your liking decreases in length and complexity. For each paid atSign you own, there is a $10 annual renewal fee. This fee does not apply to the free atSigns. What makes atSigns so awesome is that if your app is the first one someone pairs an atSign with, after buying their atSign, we pay you to commission on that. Yep, you don't pay us, we pay you.
```
Comment (cpswan) [inline content/docs/reference/pricing-and-costs.md]: drop `to` before `commission`

**Review by cpswan — CHANGES_REQUESTED** [review]
Hi @HVbajoria - thanks for the contribution. Mostly LGTM, but a couple of minor tweaks please.

**Comment by HVbajoria** [comment]
Hi @cpswan, 
Sorry for the little errors. I have corrected them. 
And for by mistakenly clicking on resolved conservation option for first suggested change.
Could you please check once and let me know if there is more scope for improvement?
Will be more than happy to learn and incorporate.
Thank you

File: content/docs/reference/synchronization.md [hunk 3]
```diff
@@ -16,18 +16,18 @@ weight: 210 # For single pages, lower is first.
 
 ## Definition
```
Comment (cpswan) [inline content/docs/reference/synchronization.md:18]: This needs to be `With atProtocol` rather than `At atProtocol`

**Review by cpswan — CHANGES_REQUESTED** [review]

**Comment by cpswan** [comment]
One down, one to go @HVbajoria

**Comment by HVbajoria** [comment]
Sorry got it.
Made the changes. 
Could you please check now and help me correct if anything is still wrong @cpswan?
Happy to learn in the process. 
Thank you

**Review by cpswan — APPROVED** [review]

**Comment by cpswan** [comment]
Thanks @HVbajoria

**Comment by HVbajoria** [comment]
Thank you @cpswan 
Happy HacktoberFest!

**Comment by JeremyTubongbanua** [comment]
@HVbajoria 
Hey! Thank you for your Hacktoberfest contribution. You could be eligible for a prize.

To be eligible, please send an email to [hacktoberfest@atsign.com](mailto:hacktoberfest@atsign.com) with a screenshot of the PR approval email (or just forward the PR approval email to us). This is to prove that your email corresponds to the correct GitHub account. Thank you!

When it comes time to distribute the prizes, we will look at all the emails we've received from that email so don't forget to do this!

**Comment by HVbajoria** [comment]
Hey @JeremyTubongbanua,
Thank you for informing me. I have send a mail with the screenshot.
Thank you

**Pull request merged**